src/deepsparse/transformers/metrics.py

```diff
@@ -36,139 +31,96 @@
 
 
 class Perplexity:
-    def __init__(self, pipeline: Pipeline, batch_size: int = 16):
+    def __init__(self, accumulate: bool = False):
```

Comment:
Could you provide an lifecycle in the docstring?

Reply:
I don't understand the comment

Reply:
Provide an example of what functions are called when, from start to finish

Reply:
Done